package-lock.json

```diff
@@ -1,8 +1,2090 @@
 {
   "name": "yt-channel-info",
-  "version": "2.2.0",
-  "lockfileVersion": 1,
+  "version": "3.0.0",
+  "lockfileVersion": 2,
```

Comment:
Might cause annoying diff war for people using npm 6

https://stackoverflow.com/questions/64813775/is-there-any-way-to-fix-package-lock-json-lockfileversion-so-npm-uses-a-specific